specification/cognitiveservices/data-plane/AzureOpenAI/authoring/stable/2022-12-01/azureopenai.json

```diff
@@ -0,0 +1,1965 @@
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The deployment has been successfully created.",
+            "schema": {
+              "$ref": "#/definitions/Deployment"
+            },
+            "headers": {
+              "Location": {
+                "description": "The location of the newly created item.",
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Creating a deployment.": {
+            "$ref": "./examples/post_deployment.json"
+          }
+        }
+      }
+    },
+    "/deployments/{deployment_id}": {
+      "get": {
+        "tags": [
+          "Deployments:"
+        ],
+        "summary": "Gets details for a single deployment specified by the given deployment_id.",
+        "operationId": "Deployments_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "deployment_id",
+            "description": "The identifier of the deployment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Deployment"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Getting a specific deployment.": {
+            "$ref": "./examples/get_deployment.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Deployments:"
+        ],
+        "summary": "Updates the mutable details of the deployment with the given deployment_id.",
+        "operationId": "Deployments_Update",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "deployment_id",
+            "description": "The identifier of the deployment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The updated values for the deployment.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentUpdate"
+            }
```

Comment:
Should this body parameter be marked as required? What is the behavior when no body is sent?